doc/man3/SSL_CONF_cmd.pod

```diff
@@ -692,6 +698,127 @@ A file or directory of certificates in PEM format whose names are used as the
 set of acceptable names for client CAs. Servers only. This option is only
 supported if certificate operations are permitted.
 
+=item B<CertValidation>
+
+The B<value> argument is a comma separated list of certificate validation
+flags to set.  If a flag string is preceded by B<-> that flag is disabled.
+See the L<X509_VERIFY_PARAM_set_flags(3)> function for more details of
+individual options.
```

Comment:
Why/how is the `-` prefix useful?  It seems that this only removes flags set earlier in the same command. Why is that necessary?